src/Orleans.Streaming.Redis/Providers/Streams/RedisStreamOptions.cs

```diff
@@ -16,6 +16,16 @@ public class RedisStreamOptions : RedisOptions
         /// normal stream pulling.
         /// </summary>
         public int QueueCacheSize { get; set; } = 1000;
+
+        /// <summary>
+        /// <para>
+        /// Whether stream messages should use the same pipeline across
+        /// deployments or should be separate between deployments.
+        /// </para>
+        /// Defaults to true
+        /// </summary>
+        public PersistenceLifetime PersistenceLifetime { get; set; } = DEFAULT_PERSISTENCE_LIFETIME;
```

Comment:
Rename to something more relevant to stream pipeline... Or something...

Reply:
Hmm, on second thought - I think this works. It _is_ streams persisting over a blue green /shrug